assemblies/features/standard/src/main/feature/feature.xml

```diff
@@ -663,12 +663,6 @@
     </feature>
 
     <feature name="ssh" description="Provide a SSHd server on Karaf" version="${project.version}">
-        <config name="org.apache.karaf.shell">
```

Comment:
This config is used by the default distribution, so I'm not sure it makes sense to remove this. I would update it instead.

Reply:
Well, shell is also used by the default distribution and it also sets this config but actually gets overridden by ssh.

Reply:
Got it, let me test the PR.